lib/train/transports/local.rb

```diff
@@ -36,6 +67,60 @@ def local?
         true
       end
 
+      def run_powershell_using_named_pipe(script)
+        script = "$ProgressPreference='SilentlyContinue';" + script
+        encoded_script = Base64.strict_encode64(script)
+        @pipe.puts(encoded_script)
+        @pipe.flush
+        JSON.parse(Base64.decode64(@pipe.readline))
+      end
+
+      def start_named_pipe_server(pipe_name)
+        require 'win32/process'
```

Comment:
Is there a reason why we are not using mixlib-shellout, since we just start a powershell command?

Reply:
I'm using `win32/process` to spawn a background process. Can I use `mixlib-shellout` to start a background process?

Since the PowerShell server script never returns (essentially a daemon) I'm not sure I can use `mixlib-shellout`.